### PR TITLE
Add: Signin and Signup Page

### PR DESCRIPTION
**Pull request by saurabhchirde**

### CODE REVIEW

**Comment by SumitSaundadkar** [comment]
code looks good to me.

File: style.css [hunk 1]
```diff
@@ -73,4 +73,8 @@ form > div > button {
 form > div > button{
```
Comment (ashishkatiyar2404) [inline style.css:73]: use classes for styling

**Review by ashishkatiyar2404 — APPROVED** [review]
your code looks good to me and go also
I have suggested a change

**Review by VLeads — APPROVED** [review]
Looks good !

File: style.css (hunk 1 above)
Comment (saurabhchirde) [reply style.css:73]: Used direct selectors in those places where creating classes is unnecessary.

**Pull request merged**